### PR TITLE
S1show collection

### PR DESCRIPTION
**Pull request by cwoodwarduk**
Story review

### CODE REVIEW

File: calumsCollection_2019-09-23.sql [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# ************************************************************
```
Comment (charliecog) [inline calumsCollection_2019-09-23.sql]: remove unwanted file

File: index.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            <div>
+                <h2>Wheelset</h2>
+            </div>
+            <div>
+                <h2>Groupset</h2>
+            </div>
+        </div>
+
+        <?php
+        $db = dbConn();
```
Comment (charliecog) [inline index.php]: move this procedural code up to the top, just have the echo down here

**Review by charliecog — COMMENTED** [review]
almost there!

File: index.php (hunk 2 above)
Comment (cwoodwarduk) [reply index.php]: Done

File: calumsCollection_2019-09-23.sql (hunk 1 above)
Comment (cwoodwarduk) [reply calumsCollection_2019-09-23.sql]: Deleted

**Comment by cwoodwarduk** [comment]
Changes made.

**Review by charliecog — APPROVED** [review]
woop!

**Pull request merged**